packages/blocks/src/root-block/edgeless/types.ts

```diff
@@ -1,5 +1,7 @@
 import type { EdgelessBlockModel } from './edgeless-block-model.js';
 
+export type EdgelessTool = BlockSuite.EdgelessToolType;
```

Comment:
Why not use `BlockSuite.EdgelessToolType` directly? 🤔

Reply:
There are many edgeless related codes use the type `EdgelessTool`. I think exporting the alias is more simple.